src/tests/ftest/control/dmg_telemetry_io_basic.yaml

```diff
@@ -0,0 +1,41 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+timeout: 150
+server_config:
+  name: daos_server
+  servers:
+    targets: 1
+pool:
+  name: daos_server
+  scm_size: 1G
+  control_method: dmg
+container:
+  type: POSIX
+  control_method: daos
+block_sizes: [10M, 500M]
+transfer_sizes: [1M]
```

Comment:
As there is only single value for transfer size, why not mention it already under `/run/ior/` and you would not have to add this extra line 69 `transfer_sizes = self.params.get("transfer_sizes", "/run/*")` in your test_telmetry_metrics method.

Reply:
I believe this comment was skipped. It is not major, but should be noted and may be implemented if you have to re-push.

Reply:
Why are we not testing multiple transfer size if the python code supports it? Maybe add another so we test at least 2?